src/app/weather-forecast/weather-forecast.component.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, OnInit} from '@angular/core';
+
+
+@Component({
+	selector: 'app-weather-forecast',
+	templateUrl: './weather-forecast.component.html',
+	styleUrls: ['./weather-forecast.component.scss']
+})
+export class WeatherForecastComponent implements OnInit {
+	
+	constructor() {}
+
+	ngOnInit() {
+		//javascript code to load current weather forecasts from openwheatermap
+		const weatherForecastLogic = `
+			window.myWidgetParam ? window.myWidgetParam : window.myWidgetParam = [];  
+			window.myWidgetParam.push({id: 11,cityid: '3170647',appid: 'd5754a8efdf1ead276cffa4f8250f1e1',units: 'metric',containerid: 'weatherForecastSpace',  });  
+			(function() {
+				var script = document.createElement('script');
+				script.async = true;
+				script.charset = "utf-8";
+				script.src = "https://openweathermap.org/themes/openweathermap/assets/vendor/owm/js/weather-widget-generator.js";
+				var s = document.getElementsByTagName('script')[0];
+				s.parentNode.insertBefore(script, s);  
+			})();
+		`;
+		//execution of the upload, in addition to an OWM js library
+		this.loadScript('https://openweathermap.org/themes/openweathermap/assets/vendor/owm/js/d3.min.js', '');
+		this.loadScript('', weatherForecastLogic);
```

Comment:
No, try to use the [`http`](https://angular.io/guide/http) module to call `api.openweathermap.org/data/2.5/weather?id=<id>&appid=<app-id>`.

The use the received object to show data in the component